documentation/DesignDocumentation.md

```diff
@@ -1,6 +1,31 @@
-# Design Documentation
+# System Design Documentation
+###### Tornelo Chess Scoresheet App
 
-### Choice of Software
+## Overview of Design Considerations
+The two main forms of design that are most relevant to the Tornelo Scoresheet app project are the System Architecture and the User Interface Design.
+
+**System Architecture**
+
+The System Architecture includes the way the model, the way the program is structured and the programming languages and tools used. 
+
+Key areas of this aspect of the system design and rationale behind relevant design choices are documented in Section 1 of this document. 
+
+**User Interface (UI) Design**
+
+The User Interface Design is a holistic approach to deciding how the applications functional elements are displayed on the screen. This includes considering the users and their experiences, as well as the necessary features and functional requirements. 
+
+Key areas of this aspect of the system design and rationale behind relevant design choices are documented in Section 1 of this document. 
+
+## 1 System Architecture
+TODO
```

Comment:
should we merge and complete these later?

Reply:
Yep, Brian will add to these sections